contracts/WildcardSteward_v3.sol

```diff
@@ -534,25 +564,30 @@ contract WildcardSteward_v3 is Initializable {
                 _foreclose(tokenId);
```

Comment:
Since this is called, we know that timeLastCollectedBenefactor would be now, meaning they have collected patronage on a foreclosed token. We therefore can calculated that overcollected amount

Reply:
Hmmm yeah definitely interesting. My thinking is that if we made that the case, we'd also have to check and account for whether the benefactor collectpatronage elsewhere, and distinguish between the two cases. This way simply always guarantees a certain behaviour

Reply:
Yeah, agreed, this is simpler to think about. Just that it does edit the `benefactorFunds` twice
.